src/mds/MDBalancer.cc

```diff
@@ -679,31 +677,31 @@ void MDBalancer::prep_rebalance(int beat)
 
     // target load
     target_load = total_load / (double)cluster_size;
-    dout(5) << "prep_rebalance:  my load " << my_load
+    dout(7) << "prep_rebalance:  my load " << my_load
```

Comment:
Above you get rid of the "prep_rebalance" prepend.  Did you want to leave it here?

Reply:
Thanks, fixed!